pkg/util/mount/mount_unsupported.go

```diff
@@ -57,3 +57,7 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 func (mounter *SafeFormatAndMount) diskLooksUnformatted(disk string) (bool, error) {
 	return true, nil
 }
+
+func IsNotMountPoint(file string) (bool, error) {
+	return true, nil
```

Comment:
true is not a bad default (vs. false), it prevents others using the mount point.